src/routes/ConsortiumManager/settings/users/general/PermissionSets/PermissionSetsCreate/PermissionSetsCreate.js

```diff
@@ -0,0 +1,145 @@
+import { useCallback, useMemo } from 'react';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { omit } from 'lodash';
+
+import { HasCommand, Layer, Loading } from '@folio/stripes/components';
+import { useShowCallout } from '@folio/stripes-acq-components';
+import { stripesConnect } from '@folio/stripes/core';
+
+import { TENANT_ID_SEARCH_PARAMS, PERMISSION_SET_ROUTES } from '../constants';
+import { PermissionSetForm } from '../../../../../../../temp';
+import { useManageTenantPermissions, useSelectPermissionSetById } from '../hooks';
+
+const PermissionSetsCreate = (props) => {
+  const { intl, stripes, history, location, match } = props;
+  const { search } = location;
+  const showCallout = useShowCallout();
+
+  const permissionSetId = match.params?.id;
```

Comment:
it should be always undefined for create, why do you need permissionSetId for create?

Reply:
not required for create, part of `PermissionSetsEdit`

Reply:
I see the naming can be confusing. Initially I was planning to use it for only Create. The component can manage Create and Edit. 
I have two options to improve.
1. Rename the component more generic Ex. "ManagePermessionSets"
2. Create a new component for Editing.
What do you think about this approaches?